src/actions/profileActions/profileActions.js

```diff
@@ -0,0 +1,49 @@
+import ACTION_TYPE from "../actionTypes";
+import axios from "axios";
+import { toast } from "react-toastify";
+import { BASE_URL } from "../../appUrls/urls";
+
+const API_URL = BASE_URL + "/api/profiles/";
```

Comment:
Add all your URL to the appUrls and simply call them here, the way the urls for create article have been done.

Reply:
I have made the changes